FAQ.md

```diff
@@ -0,0 +1,32 @@
+# Eudico FAQ
+
```

Comment:
What about adding here a note of the like of: "you can probably find many of these commands in lotus docs [link], but in this FAQ you'll find a list of some that may come handy for the typical user flows needed to interact with Eudico".